image_cropping/widgets.py

```diff
@@ -4,7 +4,12 @@
 
 import django
 from django import forms
-from django.contrib.admin.templatetags import admin_static
+try:
+    from django.contrib.admin.templatetags import admin_static
+except ImportError:
+    class admin_static:
+        def static(path):
+            return path
```

Comment:
Yes, I see that the class definition is not the most elegant solution. However, I think using templatetags in python code should be avoided as well.
According to https://docs.djangoproject.com/en/3.0/topics/forms/media/#paths-in-asset-definitions it seems that form media assets paths will be prefixed correctly. Therefore just using the plain path would be nicer.
The class was necessary to satisfy the following conditions:
- Use plain paths, without templatetags (at least for newer code)
- Keeping the code backwards compatible
- Only introduce changes around the import lines of the file (non-intrusive)

I think the best solution would be to eliminate the use of templatetags everywhere. But that would be more intrusive.
Anyway, I think we can close the issue because @payamnj merged a solution already.

Reply:
But I haven't merged a solution :)

Reply:
@saemideluxe I just liked your solution and removed my PR. I think I've tried the way that @Mogost suggested first and it failed in the test probably because `admin_static` is a class but `django.templatetags.static.static` is a function :)